app/templates/home.html

```diff
@@ -55,24 +39,19 @@
   <div class="my-container">
     <h1>What should I eat?</h1>
   </div>
-	<div class="col-md-12 text-center">
-		<div id="fb-root"></div>
-			<script>(function(d, s, id) {
-				var js, fjs = d.getElementsByTagName(s)[0];
-				if (d.getElementById(id)) return;
-				js = d.createElement(s); js.id = id;
-				js.src = 'https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.11&appId=1070406826462651';
-				fjs.parentNode.insertBefore(js, fjs);
-				}(document, 'script', 'facebook-jssdk'));
-			</script>
-			<div class="fb-login-button" data-width="300" data-max-rows="1" data-size="large" data-button-type="login_with" data-show-faces="false" data-auto-logout-link="true" data-use-continue-as="false"></div>
```

Comment:
What is the purpose in removing this JS script?

Reply:
the JS script is copied from facebook for a login button to appear on page but that login method does not work since we did not get a valid access token, so the script is redundant to keep after changing login method